apps/aqua-post-config/playbook/group_vars/all.yaml

```diff
@@ -0,0 +1,3 @@
+aqua_url: https://aqua.apps.clab.devops.gov.bc.ca
```

Comment:
Just a thought but couldn't this also be an ENV? Extra vars `-e` will take priority if set but if you use an ENV it's one less thing to pass in for k8s Job.

Also, up to you but I see you're missing a bunch of new lines at the end of your files 🕵🏻‍♂️